Tribler/Main/vwxGUI/SearchGridManager.py

```diff
@@ -534,12 +533,17 @@ def refreshGrid(self, remote=False):
 
     @call_on_reactor_thread
     def modifyTorrent(self, torrent, modifications):
+        # TODO: change this
```

Comment:
This one is gone in another branch of mine, which is a successor of this one. It's getting a little huge, so I split it into several small PRs, doing the metadata thing bit by bit.